docs/templates/plugin.rst.j2

```diff
@@ -116,11 +116,13 @@ Parameters
                     <td class="elbow-placeholder"></td>
                 {% endfor %}
                 {# parameter name with required and/or introduced label #}
-                <td colspan="@{ from_kludge_ns('maxdepth') - loop.depth0 }@">
-                    <b>@{ key }@</b>
-                    {% if value.get('type', None) %}<br/><div style="font-size: small; color: red">@{ value.type }@</div>{% endif %}
-                    {% if value.get('required', False) %}<br/><div style="font-size: small; color: red">required</div>{% endif %}
-                    {% if value.version_added %}<br/><div style="font-size: small; color: darkgreen">(added in @{value.version_added}@)</div>{% endif %}
+                <td colspan="@{ from_kludge_ns('maxdepth') - loop.depth0 }@" style="position: relative">
+                    <b>@{ key }@</b><br/>
```

Comment:
I don't think this `<br/>` is actually needed, since the following `<div>` is a block element.

Reply:
Probably not. I'll test.

Reply:
Fixed.

Reply:
Although the real fix is to use CSS. But that's another story.